### ISSUE TITLE
Bug Report Zooming causes the second line to act strange

### ISSUE THREAD

**Issue by alex3305** [microsoft/terminal#2211]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.18362.239]
Windows Terminal version (if applicable): 0.3
```

# Steps to reproduce

1. Make sure at least two lines exists in the terminal window (eg. by typing in a command)
2. Use CTRL+mouse scroll wheel to zoom either in or out

# Expected behavior

The buffer is not modified and the second line stays in place.

# Actual behavior

## 100% zoom
This is expected behaviour

### Powershell
![image](https://user-images.githubusercontent.com/5155694/62409872-07382d00-b5de-11e9-82c1-ca4936b8af94.png)

### WSL 1
![image](https://user-images.githubusercontent.com/5155694/62409874-0d2e0e00-b5de-11e9-922b-c390533e85f6.png)

## 1x zoomed in
The second line is indented

### Powershell
![image](https://user-images.githubusercontent.com/5155694/62409893-39e22580-b5de-11e9-94c3-abd1a2e17ac9.png)

### WSL 1
![image](https://user-images.githubusercontent.com/5155694/62409894-3cdd1600-b5de-11e9-9051-8ee08a5c16d4.png)

## 1x zoomed out
The second line is indented the other way, causing the data to return to the first line.

### Powershell
![image](https://user-images.githubusercontent.com/5155694/62409901-51b9a980-b5de-11e9-9c05-7d36029a9d04.png)

### WSL 1
![image](https://user-images.githubusercontent.com/5155694/62409898-4bc3c880-b5de-11e9-8e9b-1eec4037d8ff.png)

**Comment by zadjii-msft** [comment]
Well that's _insane_.

Dunno what's causing this - seems like it might be in the conpty layer. Crazy that this _doesn't_ repro for `cmd.exe`.

**Comment by zadjii-msft** [comment]
Hey there was just a massive change (#4741) in this area, and I can't seem to repro this behavior anymore. I believe the next release is going to fix this, if not by #4741 then one of the other PRs in this area. If this actually isn't fixed in 0.10 (whenever that gets released), then feel free to post here and we'll re-open this one. 

Thanks all!
